Frontend/ui-customer/src/Resources/Localizable/EN.purs

```diff
@@ -1352,3 +1352,5 @@ getEN stringKey =
         UNCOMFORTABLE_BIKE -> "Uncomfortable Bike"
         DRIVER_AVAILABLE -> "Driver available"
         DRIVERS_AVAILABLE -> "Drivers available"
+        METRO_TICKETS -> "Metro Tickets"
+        EVERY_AMOUNT_ONE_FREE amount -> "Every"<> " "<>amount <>"rd"<> " "<>  "one free"
```

Comment:
use some other word instead of amount. and `" "` can be concatenated with "one free".

Reply:
Also rename the constructor name.